infrastructure/main.tf

```
@@ -60,4 +60,51 @@ module "judicial-booking-database-v11" {
   sku_capacity       = var.database_sku_capacity
   common_tags        = var.common_tags
   postgresql_version = "11"
-}
\ No newline at end of file
+}
+
+module "judicial-booking-database-v15" {
+  source             = "git@github.com:hmcts/terraform-module-postgresql-flexible?ref=master"
+
+  providers = {
+      azurerm.postgres_network = azurerm.postgres_network
+      }
+
+  admin_user_object_id = var.jenkins_AAD_objectId
+  business_area        = "cft"
+  name               = join("-", [local.app_full_name, "postgres-db", "v15"])
+  product            = var.product-V15
+  env                = var.env
+  component          = var.component-V15
```

```suggestion
  product            = var.product
  env                = var.env
  component          = var.component
```
No need for new variables at all - can also remove those new references in `variables.tf`